docs/user/openstack/install_upi.md

```diff
@@ -336,22 +353,29 @@ values:
 
 <!--- e2e-openstack-upi: INCLUDE START --->
 ```sh
-$ python -c 'import yaml
+$ python -c 'import os
+import yaml
+os_net_id = os.getenv("OS_NET_ID")
+path = "common.yaml"
+facts = yaml.safe_load(open(path))[0]["tasks"][2]["set_fact"]
```

Comment:
Can't we find a better way to load the facts than rely on an arbitrary index? This is very fragile and might break in cryptic ways as soon as someone touches `common.yaml`. At the minimum, leave a comment in `common.yaml`.